tests/model_stats.tsv

```diff
@@ -513,3 +513,5 @@ TID_TD-H8	194
 TID_TD-H8_GMRS	7
 TID_TD-H8_HAM	2
 Baofeng_BF-888_Custom	6
+Retevis_HA1G    55400
+Retevis_HA1UV   55400
```

Comment:
This file is generated, please do not add these lines, which skew the stats incorrectly.